docs/adr/adr_monorepo.md

```diff
@@ -31,16 +31,21 @@ For a frontend and backend devs coder can work on small tasks completly separate
 Mono-Repo
 
 *Pro:*
-- The main reason to move to a monorepo set-up is to force us to communicate/coordinate with each other and to feel a shared responsibility for the monorepo. :)
+- The main reason to move to a monorepo set-up is to force us to communicate/coordinate with each other and to feel a shared responsibility for the monorepo.
 
 other side-effects
-- deployment is easier since only one script is needed.
-- management of frontend vs. backend versioning is reduced to a minimum.
+- deployment is easier since only one deployment script is needed.
+- management of frontend vs. backend versioning is reduced to a minimum. For each deployed version, you have a state at which you know that FE and BE worked together. 
+- the development set-up simplifies since only one repo needs to be cloned and React could be run from within Docker.
+- all developers are aware when master is broken. If they cannot fix it they could encourage others. 
 
 *Con:*
 Technically, there are problems that come with a Mono-repo set up: slow code search, time for testing, chaotic repo,...
-However, we argue that all the problems really surface with a much larger codebase and team.
+However, we argue that all the problems really surface when 
+- the codebase becomes so large a virtual filing system, such as VFS for Git from Microsoft.
+- we have hundreds of developers on our team.
```

Comment:
"many more developers on our team, or at least fully separate teams of 7+ people each that have dedicated ownership of specific parts of the repo"  

I imagine we will want to revisit the decision if/when we get to size of 50 people, not 100+ ;)